src/Core/tests/WebTestCase.php

```diff
@@ -0,0 +1,288 @@
+<?php
+
+namespace ForkCMS\Core\tests;
+
+use Countable;
+use Doctrine\Common\DataFixtures\Executor\ORMExecutor;
+use Doctrine\Common\DataFixtures\FixtureInterface;
+use Symfony\Bundle\FrameworkBundle\KernelBrowser;
+use Symfony\Component\DomCrawler\Crawler;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use function func_num_args;
+
+abstract class WebTestCase extends \Symfony\Bundle\FrameworkBundle\Test\WebTestCase
+{
+    protected KernelBrowser $client;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+        static::createClient();
+        static::loadFixture(...static::getClassFixtures());
+    }
+
+    /** @return FixtureInterface[] */
+    protected static function getClassFixtures(): array
+    {
+        return [];
+    }
+
+    final protected static function loadFixture(FixtureInterface ...$fixture): void
+    {
+        (new ORMExecutor(self::getContainer()->get('doctrine.orm.entity_manager')))->execute($fixture, true);
+    }
+
+    /**
+     * @param string[] $expectedContent
+     * @param array<string, mixed> $requestParameters
+     */
+    final protected static function assertPageLoadedCorrectly(
+        string $url,
+        ?string $pageTitle = null,
+        array $expectedContent = [],
+        int $httpStatusCode = Response::HTTP_OK,
+        string $requestMethod = Request::METHOD_GET,
+        array $requestParameters = []
+    ): void {
+        static::assertHttpStatusCode($url, $httpStatusCode, $requestMethod, $requestParameters);
+        if ($pageTitle !== null) {
+            self::assertPageTitleSame($pageTitle, 'Page title is not "' . $pageTitle . '".');
+        }
+        static::assertResponseHasContent(...$expectedContent);
+    }
+
+    final protected static function assertHasLink(string $text, string $url): void
+    {
+        static::assertMinCount(
+            1,
+            static::getCrawler()->filter('a:contains("' . $text . '")[href="' . $url . '"]')
+        );
+    }
+
+    public static function assertMinCount(int $expectedCount, Countable|iterable $haystack, string $message = ''): void
```

Comment:
dded this one because I'm expecting at least 3 errors for instance, but then on the sumo fork there is like a fourth required field it would fail. I normally also check that the error messages I'm expected are visible.
But it might be better that if a fourth error message is shown the test actually should fail, in that case we don't need this one

Reply:
I just find it somewhat confusing as it is just a oneliner. So it seems to be just an alias. 
Or something that is not really needed and only more "magic" that is not really documented, while all methods from PHPUnit are well documented and can be found more easily.

But I do understand it from a developer point of view. So fine by me.